dockerfiles/Dockerfile.openvino

```diff
@@ -60,6 +59,10 @@ RUN git clone --recursive -b $ONNXRUNTIME_BRANCH $ONNXRUNTIME_REPO /onnxruntime
     /bin/sh onnxruntime/dockerfiles/scripts/install_common_deps.sh && \
```

Comment:
in line 60, it's missing / in front of onnxruntime/dockerfiles/scripts/install_common_deps.sh
the source was cloned to /onnxruntime in previous line.

Reply:
addressed